opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/MessagesControllerTest.java

```diff
@@ -72,6 +84,67 @@ protected Application configure() {
         return new ResourceConfig(MessagesController.class);
     }
 
+    // Allow entity body for DELETE method on the client side.
+    @Override
+    protected void configureClient(ClientConfig config) {
+        config.property(ClientProperties.SUPPRESS_HTTP_COMPLIANCE_VALIDATION, true);
```

Comment:
per https://stackoverflow.com/questions/36310863/jersey-test-http-delete-method-with-json-request and https://stackoverflow.com/questions/25229880/how-to-send-enclose-data-in-delete-request-in-jersey-client